.gitignore

```diff
@@ -43,3 +43,6 @@ composer.lock
 .Trashes
 ehthumbs.db
 Thumbs.db
+composer.lock
+.phpunit.result.cache
+phpunit.xml
```

Comment:
because the only lock file that's important is the one in your project because you won't be installing dependencies in the package. Here i will install multiple versions for testing. For instance when we run travis tests and install different versions of Laravel like
https://github.com/musonza/chat/blob/802d03566a40c9b26fab5130c535f9b77e6c1463/.travis.yml#L3-L8

Reply:
Ah doh! Yeah that's right